environ/compat.py

```diff
@@ -33,7 +33,9 @@ class ImproperlyConfigured(Exception):
     DJANGO_POSTGRES = 'django.db.backends.postgresql'
 
 # back compatibility with redis_cache package
-if find_loader('redis_cache'):
+if DJANGO_VERSION is not None and DJANGO_VERSION >= (4, 0):
+    REDIS_DRIVER = 'django.core.cache.backends.redis.RedisCache'
+elif find_loader('redis_cache'):
```

Comment:
if a user has redis cache, they'll likely want to use it here (or for backwards compatibility sake). I'd switch the order to check whether the user already has redis cache or not.